docs/csharp/language-reference/compiler-messages/toc.yml

```diff
@@ -273,3 +273,5 @@
     href: cs1591.md
   - name: Compiler Warning (level 4) CS1610
     href: cs1610.md
+  - name: Compiler warning (level 5) CS8892
```

Comment:
I started to use sentence-casing as it's now the recommended styleguide. Is it okay to be inconsistent with the other toc nodes?

Reply:
Thanks for asking this.

I think it looks jarring to mix them. What do you think of updating this section of the TOC to match the current style guide?